### PR TITLE
docs: more Svelte 5 conversion

### PR DESCRIPTION
**Pull request by dummdidumm**
closes https://github.com/sveltejs/svelte.dev/issues/549

fixes some leftovers, adds a few callouts to legacy syntax

### CODE REVIEW

File: documentation/docs/98-reference/26-$lib.md [hunk 1]
```diff
@@ -2,4 +2,18 @@
 title: $lib
 ---
 
-TODO
+As a convenience, SvelteKit makes imports from `src/lib` (if you didn't adjust it in your [config file](configuration#files)) available via the `$lib` alias.
```
Comment (ghostdevv) [inline documentation/docs/98-reference/26-$lib.md:5]:
```suggestion
SvelteKit automatically makes files under `src/lib` available using the `$lib` import alias. You can change which directory this alias points to in your [config file](configuration#files).
```

This maybe makes it a bit clearer?

File: documentation/docs/20-core-concepts/30-form-actions.md [hunk 2]
```diff
@@ -154,6 +151,9 @@ export const actions = {
 {/if}
 ```
 
+> [!LEGACY] In Svelte 4
+> In Svelte 4, you'd use `export let form` instead to declare a property
```
Comment (eltigerchino) [inline documentation/docs/20-core-concepts/30-form-actions.md:155]:
```suggestion
> In Svelte 4, you'd use `export let data` and `export let form` instead
```

**Review by eltigerchino — DISMISSED** [review]

File: documentation/docs/20-core-concepts/20-load.md [hunk 3]
```diff
@@ -85,8 +88,8 @@ export async function load() {
 ```svelte
 <!--- file: src/routes/blog/[slug]/+layout.svelte --->
 <script>
-	/** @type {import('./$types').LayoutData} */
-	export let data;
+	/** @type {{ data: import('./$types').LayoutData }} */
+	export let { data } = $props();
```
Comment (SymphonySimper) [inline documentation/docs/20-core-concepts/20-load.md:92]:
```suggestion
	let { data } = $props();
```

**Comment by SymphonySimper** [comment]
https://github.com/sveltejs/svelte.dev/issues/549#issuecomment-2436859526

> Couple of sections still seems to use <slot></slot> | <slot /> instead of {@render children()}.
> 
> https://svelte.dev/docs/kit/routing
> https://svelte.dev/docs/kit/load#Layout-data
> https://svelte.dev/docs/kit/advanced-routing

Can this be addressed as well?

**Comment by eltigerchino** [comment]
> [sveltejs/svelte.dev#549 (comment)](https://github.com/sveltejs/svelte.dev/issues/549#issuecomment-2436859526)
> 
> > Couple of sections still seems to use `<slot></slot>` | `<slot />` instead of `{@render children()}`.
> > [svelte.dev/docs/kit/routing](https://svelte.dev/docs/kit/routing)
> > [svelte.dev/docs/kit/load#Layout-data](https://svelte.dev/docs/kit/load#Layout-data)
> > [svelte.dev/docs/kit/advanced-routing](https://svelte.dev/docs/kit/advanced-routing)
> 
> Can this be addressed as well?

Yes, these also contain `export let data`. Good find.

**Comment by SymphonySimper** [comment]
> > [sveltejs/svelte.dev#549 (comment)](https://github.com/sveltejs/svelte.dev/issues/549#issuecomment-2436859526)
> > > Couple of sections still seems to use `<slot></slot>` | `<slot />` instead of `{@render children()}`.
> > > [svelte.dev/docs/kit/routing](https://svelte.dev/docs/kit/routing)
> > > [svelte.dev/docs/kit/load#Layout-data](https://svelte.dev/docs/kit/load#Layout-data)
> > > [svelte.dev/docs/kit/advanced-routing](https://svelte.dev/docs/kit/advanced-routing)
> > 
> > 
> > Can this be addressed as well?
> 
> Yes, these also contain `export let data`. Good find.

Thanks! All most all of the `export let data` has been fixed. The only one I could find is at [`90-adapter-vercel.md`](https://github.com/sveltejs/kit/blob/b12876bf32f2a7f50f069f268bbde42cc9bb7fe4/documentation/docs/25-build-and-deploy/90-adapter-vercel.md?plain=1#L144).

**Comment by SymphonySimper** [comment]
https://github.com/sveltejs/kit/blob/b12876bf32f2a7f50f069f268bbde42cc9bb7fe4/documentation/docs/20-core-concepts/20-load.md?plain=1#L127

Shouldn't this be `index + 1`  for *next*? And the `;` should be outside of the brackets. 
**Suggested Change**:
```patch
- let next = $derived(data.posts[index - 1];)+++
+ let next = $derived(data.posts[index + 1]);+++
```

**Review by eltigerchino — APPROVED** [review]

**Comment by eltigerchino** [comment]
This PR fixes the lint errors for `src/exports/vite/public.d.ts` too.

**Pull request merged**